### DISCUSSION TITLE
Inconsistent display of color tagged text.

### DISCUSSION THREAD

**Discussion by lIllIIlI** [FakeFishGames/Barotrauma#14337] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Some GUI displays proper color for items tagged with a specific color via translation xml, i.e: 
  <entityname.Patchouli_gear>‖color:120,75,100,255‖帕秋莉装束‖color:end‖</entityname.Patchouli_gear>
  <entityname.Patchouli_gear>‖color:120,75,100,255‖Patchouli's Outfit‖color:end‖</entityname.Patchouli_gear>
Appears correctly in tooltips, but not in crafting / shop menus.
![Barotrauma_HewB5xlmMx](https://github.com/user-attachments/assets/a713ef62-e227-40c3-aa01-c715f537e039)

Similarly, chat text popups do not adhere to the color tags while the chat window does.
![image](https://github.com/user-attachments/assets/ca7dc18a-7ef9-45a8-acac-70ab51aeccac)
![image](https://github.com/user-attachments/assets/7eacd612-1aa2-4089-a2d2-45e1dcdceff0)

I'd like the colors to be supported in all types of texts.

### Reproduction steps

1. Use localization text file to give item names color using ||color:###,###,###,###||<Item Name>||color:end||
2. color appears properly in tool tips, chat window, fabrication tooltip
3. color does not appear but the code itself appears in chat bubbbles and fab/shop menus

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by lIllIIlI** [comment]
Not fixed in unstable 😭 
![image](https://github.com/user-attachments/assets/cc9ac9d7-a607-4f2f-b2fd-d1118ab9fc0e)

**Reply by Regalis11** [reply]
The fix hasn't made it to unstable I'm afraid, we still need to review and test the fix internally first. Sorry for the inconvenience!

**Comment by Jasontti** [comment]
Tested a fix for this, no issues found.
Used the example above with modified revolver name:
`<entityname.revolver>‖color:120,75,100,255‖revolver‖color:end‖</entityname.revolver>`
![image](https://github.com/user-attachments/assets/5cf00e0a-9aca-4b62-89b8-1e96255bb32c)

Closing discussion as resolved.

**Reply by lIllIIlI** [reply]
The colors in inventory always worked. The issue was the color code being shown in shop menus, fabricator, Sub editor item list, as well as text in chat bubbles. Are those all resolved?

**Reply by Jasontti** [reply]
Checked every place mentioned in original post, but seems that i missed the name in sub editor menu (hover text is still fine)
![image](https://github.com/user-attachments/assets/6a59725c-2707-4220-a603-480729120e89)

**Comment by Jasontti** [comment]
Opening discussion.
Missed sub editors name showing "color:" part in the menu, when testing.
![image](https://github.com/user-attachments/assets/822ade85-f382-447e-9a9a-078306a75af7)

**Reply by Regalis11** [reply]
Should be fixed now
